WMIAdventure/frontend/src/js/components/profile/organisms/ChangeDeckCard/ChangeDeckCard.js

```diff
@@ -116,35 +111,18 @@ class ChangeDeckCard extends React.Component {
     }
 
 
-    onNewCardChoose = (event, id, name, subject, tooltip, image, levels, access) => {
+    onNewCardChoose = (event, id, access) => {
         if (!access)
             return;
-
-        // TODO: If we will have levels implemented change this
-        const description = this.state.allCards.filter(c => c.id === id)[0].levels[0].effects_description;
+        const chosenCard = this.state.allCards.filter(card => card.id === id)[0];
```

Comment:
Przyda to mi się do dokończenia taska, bo tamta stara logika trochę jest dla mnie problematyczna. Widzę że tutaj będzie obecny poziom jako *level*, a przydałyby mi się do mojego PR jeszcze wszystkie możliwe levele danej karty w jakimś propsie. Chyba że do tego w jakimś innym miejscu mam się dokopać. Sam nie wiem.

Reply:
dodałem `next_level_cost` tak jak u Ciebie w pull requeście